app/controllers/api_controller.rb

```diff
@@ -2,4 +2,25 @@
 class ApiController < ActionController::API
   include Response
   include ExceptionHandler
+
+  def render_resource(resource)
+    if resource.errors.empty?
+      render json: resource
+    else
+      validation_error(resource)
+    end
+  end
+
+  def validation_error(resource)
```

Comment:
How about we make this a private method since it's only called in `render_resource `?